app/views/aspect_memberships/_aspect_membership_dropdown.mobile.haml

```diff
@@ -1,5 +1,5 @@
 %div
-  %select{:name => 'user_aspects', :class => 'user_aspects', 'data-person-id' => @person.id}
+  %select{:name => 'user_aspects', :class => 'user_aspects form-control', 'data-person-id' => @person.id}
```

Comment:
Then you keep the old syntax, for the entire hash, don't mix the syntaxes. Though here you could write: `data: {"person-id" => @person.id}`